Makefile

```diff
@@ -96,7 +96,7 @@ binaries: manager clusterawsadm ## Builds and installs all binaries
 
```

Comment:
`LDFLAGS` variable is defined on line 236, can we move it before the `Help` section now that it's being used in multiple places?

Reply:
@vincepri moved `LDFLAGS` up in the Makefile before the `Help` section.